server/neptune/workflows/internal/pr/revision/processor.go

```diff
@@ -60,7 +60,11 @@ func (p *Processor) Process(ctx workflow.Context, prRevision Revision) {
 			}
 		}
 	}
-	p.PolicyHandler.Process(ctx, failedPolicies)
+	if len(failedPolicies) > 0 {
+		workflow.Go(ctx, func(c workflow.Context) {
```

Comment:
Remind me why this is run in another go routine again?

Reply:
Good callout, forgot for a minute that the revision processor itself already ran as a separate goroutine (originally added that there bc I wanted to make sure policy handler didn't block the workflow from receiving and handling new revisions. Will remove